src/OpenTelemetry.Exporter.Console/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+* Changed error handling, `ConsoleExporter` will only output an error message
```

Comment:
The description doesn't seem correct to me.

Reply:
Update it, please help to check my English